tutorials/plot_eeg_no_mri.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+"""
+EEG forward operator with a template MRI
+========================================
+
+This tutorial explains how to compute the forward operator from EEG data
+using the standard template MRI subject ``fsaverage``.
+
+.. important:: Source reconstruction without an individual T1 MRI from the
+               subject will be less accurate. Do not over interpret
+               activity locations which can be off by multiple centimeters.
+
+.. contents:: This tutorial covers:
+   :local:
+   :depth: 2
+
+"""
+
+# Authors: Alexandre Gramfort <alexandre.gramfort@inria.fr>
+#          Joan Massich <mailsik@gmail.com>
+#
+# License: BSD Style.
+
+import os.path as op
+
+import mne
+from mne.datasets import set_montage_coreg_path, fetch_fsaverage, sample
+
+# Convenience function to set ``subjects_dir`` default value for users who
+# only ever plan to do montage-based coreg with fsaverage.
+subjects_dir = set_montage_coreg_path()
+
+# Download fsaverage files
+fs_dir = fetch_fsaverage(verbose=True)
```

Comment:
For users with `fsaverage` already, they will get theirs updated to have the latest files. On CircleCI, which sets `SUBJECTS_DIR` to the sample dataset directory, this looks like:
```
438 files missing from fsaverage in /home/circleci/mne_data/MNE-sample-data/subjects/fsaverage
Downloading missing files remotely
Downloading https://files.osf.io/v1/resources/rxvq7/providers/osfstorage/5cb759eea3bc970017f17d50?action=download&version=1&direct&revision=1 (536.0 MB)

Verifying hash 614a3680dcfcebd5653b892cc1234a4a.
File saved as /tmp/tmpihd8a9hb/fsaverage.zip.

Extracting missing files
Successfully extracted 438 files
```

Reply:
@larsoner to make both you and us (with @massich) happy I would suggest to create a function

mne.datasets.fsaverage.data_path()

that uses set_montage_coreg_path and fetch_fsaverage internally. The reason is that EEG people using a template should not bother about the subjects_dir and freesurfer jargon.

also you switched the example to use sample which uses fif files. I had proposed to use eegbci because it's edf files which are standard for EEG people. It feels more natural.

Any objection to these 2 points @larsoner ?

Reply:
> mne.datasets.fsaverage.data_path()

Maybe just `mne.datasets.fetch_fsaverage(subjects_dir=None, update_path=True)`, and `update_path=True` will set the path using the `set_montage...` function. And we can even then make the `set_montage...` thing private.

I prefer the `fetch_` nomenclature here instead of `data_path` since it does something closer to what the existing `fetch_` functions do than to what the existing `data_path` functions do.

> I had proposed to use eegbci because it's edf files which are standard for EEG people. It feels more natural.

Above you said ["We should use sample here"](https://github.com/mne-tools/mne-python/pull/6146#discussion_r276330433) which is why I switched to the `sample` dataset. EEGBCI is fine with me, though showing a field map is a particularly uninteresting example, and something users are unlikely to want to do. It would be much better to, say, compute some classifier and then project it to source space or something. I can do it at the sprint next week.